lib/fixtures/set-triggers.sql

```diff
@@ -0,0 +1,24 @@
+-- Delete triggers
+DROP TRIGGER IF EXISTS toggleOrgUsers on organisations;
+
+-- Create procedures
+CREATE OR REPLACE FUNCTION toggleOrgUsers()
+RETURNS TRIGGER AS
+$BODY$
+  BEGIN
+    UPDATE people
+    SET active = NOT people.active
+    WHERE org_id = OLD.id;
+
+    RETURN NEW;
+  END;
+$BODY$
+LANGUAGE plpgsql;
+
+-- Create triggers
+CREATE TRIGGER toggleOrgUsers
+AFTER
+UPDATE ON organisations
+FOR EACH ROW
+WHEN (OLD.active IS DISTINCT FROM NEW.active)
+EXECUTE PROCEDURE toggleOrgUsers();
```

Comment:
from this comment we should go with whatever is easiest: https://github.com/emfoundation/ce100-app/issues/509#issuecomment-269489413

I think the easiest is to make this function run only when the org is being deactivated.